### PR TITLE
M3 3266 e2e syntax update

### PR DESCRIPTION
**Pull request by jstamis-linode**
## Description

Updated syntax on page object files for webdriverIO V5 
linted files

## Type of Change
- Non breaking change ('update', 'change')

If the any above types of change apply to this pull request, please ensure to include one of the listed keywords in the pull request title.

## Applicable E2E Tests

To run relevant E2E tests, run these commands in 3 separate terminals:

1. `yarn && yarn start`
2. `yarn selenium`
3. `yarn e2e --spec=COMMA_SEPARATED_LIST_OF_SPECS_HERE --browser=headlessChrome`

## Note to Reviewers
Tests are still going to fail for now as the next step is updating the spec files. Once the spec files have been updated we will begin to prune e2e tests we no longer feel provide value.

### CODE REVIEW

**Comment by jstamis-linode** [comment]
It looks like I didn't squash everything, I will merge and squash when approved.

**Comment by patthiel** [comment]
just leaving a note to inform i have begun to review this

File: packages/manager/e2e/pageobjects/account/user-detail.page.js [hunk 1]
```diff
@@ -1,39 +1,78 @@
 const { constants } = require('../../constants');
+const { assertLog } = require('../../utils/assertionLog');
 
 import Page from '../page';
+import { browserCommands } from '../../config/custom-commands';
```
Comment (patthiel) [inline packages/manager/e2e/pageobjects/account/user-detail.page.js]: unused import, safe to remove

File: packages/manager/e2e/pageobjects/account/users.page.js [hunk 2]
```diff
@@ -1,120 +1,194 @@
...
+      .toBe(true);
+  }
+
+  add(userConfig) {
+    this.addUserButton.waitForDisplayed(constants.wait.normal);
+    this.addUserButton.click();
+    this.drawerTitle.waitForText(constants.wait.normal);
+    this.notice.waitForDisplayed(constants.wait.normal);
+    //TODO check if we need to use
+    // browser.trySetValue(`${this.createDrawerEmail.selector}`, userConfig.email):
```
Comment (patthiel) [inline packages/manager/e2e/pageobjects/account/users.page.js]: looks like setvalue is working here and we can remove these comments?

**Comment by patthiel** [comment]
related tweet: https://twitter.com/iamdevloper/status/851445899008106496

File: packages/manager/e2e/pageobjects/account/user-detail.page.js (hunk 1 above)
Comment (jstamis-linode) [reply packages/manager/e2e/pageobjects/account/user-detail.page.js]: Will remove this in the next PR for smoke tests

File: packages/manager/e2e/pageobjects/account/users.page.js (hunk 2 above)
Comment (jstamis-linode) [reply packages/manager/e2e/pageobjects/account/users.page.js]: Will remove this in the next PR.

**Pull request merged**